simulator/build.gradle.kts

```diff
@@ -91,3 +91,29 @@ tasks.register<Copy>("untarTestBlockStream") {
 tasks.named("processResources") { dependsOn(tasks.named("untarTestBlockStream")) }
 
 tasks.named("sourcesJar") { dependsOn(tasks.named("untarTestBlockStream")) }
+
+// Task to prepare and run Docker container
+tasks.register("startSimulatorDockerContainers") {
+    description = "Prepare and run the simulator containers in publisher and consumer mode"
+    group = "docker"
+
+    doLast {
```

Comment:
why is there `doLast` here?

Reply:
I want to make sure that those steps are the last thing we do in the whole lifecycle. more info here https://docs.gradle.org/current/userguide/build_lifecycle.html